.travis.yml

```diff
@@ -18,6 +18,7 @@ script:
   - cargo test --features embed-lgpl-docs --jobs 1
   # catch any sneaked in lgpl docs
   - cargo build --features purge-lgpl-docs --jobs 1
+  - cargo doc --features "dox"
```

Comment:
Better move this one line down, as it middle of `# catch any sneaked in lgpl docs` part